larq/models.py

```diff
@@ -230,14 +230,9 @@ def generate_table_row(self, table_config):
 
     def _get_bitwidth(self, weight):
         try:
-            for quantizer, quantized_weight in zip(
-                self._layer.quantizers, self._layer.quantized_latent_weights
-            ):
-                if quantized_weight is weight:
-                    return quantizer.precision
+            return weight.precision
         except:
```

Comment:
```suggestion
         except AttributeError:
```

Assuming this'd be an `AttributeError`, I think it's better to catch it more explicitly.

Reply:
Or maybe:

```python 
    def _get_bitwidth(self, weight):
        return getattr(weight.precision, 32)
```

Reply:
Fixed in 2f79a608529764e6c7bd944deeb44aa4268155c7